web/build.sh

```diff
@@ -14,565 +14,99 @@ THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BA
 
 . "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
 
+WORKING_DIRECTORY=`pwd`
+
```

Comment:
```suggestion
```

This shouldn't be necessary (also in 6 other locations) as build scripts should never be dependent on the working directory from where they are run.